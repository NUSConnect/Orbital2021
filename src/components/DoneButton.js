import React from "react";
import { Text, TouchableOpacity } from "react-native";

export default function DoneButton({ ...props }) {
    return (
        <TouchableOpacity style={{ marginHorizontal: 10 }} {...props}>
            <Text style={{ fontSize: 16 }}>Done</Text>
        </TouchableOpacity>
    );
}
