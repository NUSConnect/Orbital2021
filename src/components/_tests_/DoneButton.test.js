import * as React from 'react'
import DoneButton from '../CreateComment'
import renderer from 'react-test-renderer'

jest.useFakeTimers()
jest.mock('react-native-vector-icons')

it('renders correctly', () => {
  const tree = renderer.create(<DoneButton />)
  expect(tree).toMatchSnapshot()
})
