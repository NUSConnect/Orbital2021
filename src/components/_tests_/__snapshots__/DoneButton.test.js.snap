// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#ffffff",
      "elevation": 3,
      "flexDirection": "row",
      "justifyContent": "center",
      "padding": 2,
      "paddingHorizontal": 5,
    }
  }
>
  <TextInput
    allowFontScaling={true}
    autoCorrect={false}
    maxLength={2000}
    multiline={true}
    onBlur={[Function]}
    onFocus={[Function]}
    placeholder="Add a comment"
    rejectResponderTermination={true}
    style={
      Object {
        "borderRadius": 10,
        "flex": 1,
        "fontSize": 16,
        "height": 40,
        "margin": 5,
        "paddingHorizontal": 15,
      }
    }
    underlineColorAndroid="transparent"
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID="send"
  />
</View>
`;
